docs/data/date-pickers/custom-components/PaperComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import Paper, { PaperProps } from '@mui/material/Paper';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
+import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker';
+
+const CustomPaper = React.forwardRef(function CustomPaper(
+  props: PaperProps,
+  ref: React.Ref<HTMLDivElement>,
+) {
+  // @ts-ignore
+  const { ownerState, ...other } = props;
+
+  return <Paper {...other} ref={ref} elevation={2} square />;
```

Comment:
One could say that this demo should instead be the `ComponentProps`... 🤔 

As for this one, maybe we could actually build a custom plain `div` or styled component?

Reply:
To be honest I don't think anyone should override this component as it is today :laughing:

Reply:
Why not? Maybe someone wants to add some text or icon of their choosing? 🤔 
For example:

```tsx
const CustomPaperWrapper = styled('div')(({ theme }) => ({
  backgroundColor: theme.palette.background.paper,
  border: `2px solid ${theme.palette.divider}`,
}));
...
const CustomPaper = React.forwardRef(function CustomPaper(
  props: PaperProps,
  ref: React.Ref<HTMLDivElement>,
) {
  // @ts-ignore
  const { ownerState, children, ...other } = props;

  return (
    <CustomPaperWrapper {...other} ref={ref}>
      {children}
      <Typography sx={{ ml: 2, mb: 2 }}>© 2222 Acme Inc.</Typography>
    </CustomPaperWrapper>
  );
});
```